e2e/cypress/integration/messaging/tooltip_visual_verification_spec.js

```diff
@@ -45,21 +26,23 @@ describe('M18697 - Visual verification of tooltips on post hover menu', () => {
         cy.postMessage('Test');
     });
 
-    it('Check dotmenu icon tooltip', () => {
+    it('M18697 - Visual verification of tooltips on post hover menu', () => {
         cy.getLastPostId().then((postId) => {
-            hoverOverItem(postId, 'button', '#dotmenu-icon-tooltip', 'More Actions');
-        });
-    });
+            verifyToolTip(postId, `#CENTER_button_${postId}`, 'More Actions');
 
-    it('Check reaction icon tooltip', () => {
-        cy.getLastPostId().then((postId) => {
-            hoverOverItem(postId, 'reaction', '#reaction-icon-tooltip', 'Add Reaction');
-        });
-    });
+            verifyToolTip(postId, `#CENTER_reaction_${postId}`, 'Add Reaction');
 
-    it('Check comment icon tooltip', () => {
-        cy.getLastPostId().then((postId) => {
-            hoverOverItem(postId, 'commentIcon', '#comment-icon-tooltip', 'Reply');
+            verifyToolTip(postId, `#CENTER_commentIcon_${postId}`, 'Reply');
         });
     });
+
+    function verifyToolTip(postId, targetElement, label) {
+        cy.get(`#post_${postId}`).trigger('mouseover');
+
+        cy.get(targetElement).trigger('mouseover', {force: true});
+        cy.findByText(label).should('be.visible');
```

Comment:
Looks like for all three case ("More Actions", "Add Reaction" and "Reply") the tooltip text is not being found - fails with `Error: Unable to find an element with the text:`

Reply:
Not sure why this passing consistently on local and cypress server.  Could you please investigate?  Did the tooltip not showed up in all cases?